scripts/run_e2e_tests.py

```diff
@@ -353,64 +398,79 @@ def run_component_template_e2e_test(ctx: Context, template_dir: str) -> bool:
     is_flag=True,
     help="Run tests in 'e2e_flaky' instead of 'e2e'.",
 )
-@click.option(
-    "-t",
-    "--test",
-    help="Run a specific test, e.g. '--test st_markdown'",
-    type=str,
-)
+@click.argument("tests", nargs=-1)
```

Comment:
Since we can't attach a help string to `@click.argument`, could you add a help string to the `@click.command` that says something like:

> Run Streamlit e2e tests. If specific tests are specified, only those tests will be run. If you don't specify specific tests, all tests will be run.

So that usage of the "tests" argument is more clear?